### PR TITLE
Revamped scheduleWorkstationWakeUpIfNecessary method.

### PR DESCRIPTION
**Pull request by xrtariq2594**
## Description of changes

I removed the "if (actionId.equals(WORKSTATION_PROCESSING))" check starting from line 65 as the event handler already takes care of that thanks to the method signature.

I also revamped the scheduleWorkstationWakeUpIfNecessary method to use differing IDs and refresh all awaiting actions. I'll have to further test the latter to ensure that it's really necessary.

Note: This requires that https://github.com/MovingBlocks/Terasology/pull/2356 is downloaded to properly test.

### CODE REVIEW

**Comment by Cervator** [comment]
Just for reference this PR (actually a couple more commits through the fluid branch) was in use with the second test server today. No issues noted although I didn't craft very far, just validated that the regular workbench could be used.

I also realized I had been using said branch locally for probably 2-3 weeks straight so far, without noticing.

File: src/main/java/org/terasology/workstation/system/WorkstationAuthoritySystem.java [hunk 1]
```diff
@@ -83,21 +83,20 @@ public void finishProcessing(DelayedActionTriggeredEvent event, EntityRef workst
         executingProcess = true;
         try {
             String actionId = event.getActionId();
-            if (actionId.equals(WORKSTATION_PROCESSING)) {
+            //if (actionId.equals(WORKSTATION_PROCESSING)) {
```
Comment (skaldarnar) [inline src/main/java/org/terasology/workstation/system/WorkstationAuthoritySystem.java]: Should be removed entirely before merging.

**Comment by Cervator** [comment]
Bump. Just realized this is still outstanding, although we've merged the engine PR.

@xrtariq2594 should we go ahead and merge? Any final tweaks like the removal of commented out stuff above?

**Comment by Cervator** [comment]
Merged with the commented code removed along with a few other style fixes and additional commits from the fluid branch. Checked on JS and it seemed fine including fluid use in a sifting table.

**Pull request merged**